datasets/me/acp_peps/crawler.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise, OccupancyStatus
+from zavod.shed.trans import (
+    apply_translit_full_name,
+    make_position_translation_prompt,
+)
+
+TRANSLIT_OUTPUT = {
+    "eng": ("Latin", "English"),
+}
+POSITION_PROMPT = prompt = make_position_translation_prompt("cnr")
+
+
+def get_latest_date(dates):
+    if dates:
+        # Convert string dates to datetime objects
+        dates = [datetime.strptime(item["datum"], "%Y-%m-%d") for item in dates]
+        latest_date = max(dates)
+        return latest_date.strftime("%Y-%m-%d")
+    return None
+
+
+def crawl_person(context: Context, person):
+    name = person.pop("imeIPrezime")
+    position = person.pop("nazivFunkcije")
+    dates = person.pop("izvjestajImovine")
+    latest_date = get_latest_date(dates)
```

Comment:
do you want to see if within a couple of hours you can:

- take the latest report id e.g. `stariIzvjestaj: 84990`
- request the CSV dump of the report using the ID https://portal.antikorupcija.me:9343/acamPublic/izvestajDetailsCSV.json?izvestajId=84990
- open the public functions file `csv_funkcije_funkcionera_....csv` (https://github.com/opensanctions/opensanctions/blob/main/datasets/_global/icij_offshoreleaks/crawler.py#L45)
- try and generate a position and occupancy using full start/end dates for each entry in that?

Reply:
it looks like this wont work for everyone so maybe just do a test of what proportion we can get these data IDs for before doing a lot of work on the zip and csv 
![image](https://github.com/user-attachments/assets/b7ee8c92-9a7c-4196-9636-a2d3704172b5)

Reply:
the family members data is also juicy - they get the RCA topic